src/docs/product/accounts/sso/azure-sso.mdx

```diff
@@ -57,7 +57,17 @@ description: "Learn about the Azure Active Directory Single Sign-On (SSO)."
     - First Name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname`
 
     - Last Name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname`
+    
+    - For more details about mappings for custom configs, see [The Role of Claims](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/technical-reference/the-role-of-claims).
 
    ![Map Identity with provider attributes](azure-map-identity-provider-attributes.png)
+   
+1. Sentry will attempt to authenticate and link your account with Azure. After successful authentication, you'll be redirected to Sentry's SSO configuration page.
 
-   For more details about mappings for custom configs, see [The Role of Claims](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/technical-reference/the-role-of-claims).
+   - The `Login URL` will be used for SP-initiated SSO. You can share it with the users in your organization.
+   
+   - Scroll down to the bottom. Ensure that `Require SSO` is checked if you want to enforce logging in with Okta.
+   
+   - Set a `Default Role` for new SSO users. Selecting `Member` should cover most use-cases.
+   
+   - If you made changes, click **Save Settings** and that's all for the setup!
```

Comment:
Without seeing this page, I've made the assumption that these are field labels that you're talking about so they'd take the UI element styling. 

```suggestion
   - You can share the "Login URL" value, which will be used for SP-initiated SSO, with the users in your organization.
   
   - Scroll down to the bottom and ensure that "Require SSO" is checked if you want to enforce logging in with Okta.
   
   - Set a "Default Role" for new SSO users. Selecting "Member" should cover most use-cases.
   
   - If you made changes, click "Save Settings" to complete your setup.
```